src/main/config.ts

```diff
@@ -112,6 +116,7 @@ export const defaults: AppStore = {
   backupList: {},
   clientId: uuid.v4(),
   preferedLanguage: '',
+  virtualdriveWindowsLetter: 'I',
```

Comment:
I could be totally wrong. But I think that when the user already have a config.json file with the stored values if you add a new entry it's default value will not be added and will return undefined until is `set`

Reply:
I chequed and it worked good, anyways I've added a default value just in case